Sources/PackageRegistry/SignatureValidation.swift

```diff
@@ -227,7 +228,7 @@ struct SignatureValidation {
                     completion(.failure(RegistryError.invalidSignature(reason: reason)))
                 case .certificateInvalid(let reason):
                     completion(.failure(RegistryError.invalidSigningCertificate(reason: reason)))
-                case .certificateNotTrusted(let signingEntity):
+                case .certificateNotTrusted(let package, let signingEntity):
```

Comment:
Do we have to add `package` to `certificateNotTrusted`? Don't we already get it as method argument (L199)?